integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -157,6 +189,31 @@ private void upgradeOperator() throws Exception {
     testClusterScaling(operator, domain, false);
   }
 
+  /**
+   * Upgrades operator to develop branch by uninstalling old release operator
+   * and installing latest operator.
+   * @throws Exception when upgrade fails or basic usecase testing or scaling fails.
+   */
+  private void upgradeOperatorFrom2_6_0() throws Exception {
+    operator.destroy();
+    // Thread.sleep(30 * 1000);
+    Map<String, Object> operatorMap = createOperatorMap(getNewSuffixCount(), true, "");
+    operatorMap.put("namespace", OP_NS);
+    operatorMap.put("releaseName", OP_DEP_NAME);
+    operatorMap.put("serviceAccount", OP_SA);
+    operatorMap.put("externalRestEnabled", true);
+    operatorMap.put("javaLoggingLevel", "FINE");
+    List<String> domNs = new ArrayList<String>();
+    domNs.add(DOM_NS);
+    operatorMap.put("domainNamespaces", domNs);
+    operator = TestUtils.createOperator(operatorMap, Operator.RestCertType.LEGACY);
+
+    checkCrdVersion();
+    checkDomainNotRestarted();
```

Comment:
For 2.6 to 3.0 upgrade, I do expect the domain to be restarted.

Reply:
Yes, domain is getting restarted. Removed the check.